### PR TITLE
Create Blog “comparing-to-kill-a-mockingbird-to-its-sequel-with-apache-spark”

### PR DESCRIPTION
**Pull request by burchems**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/comparing-to-kill-a-mockingbird-to-its-sequel-with-apache-spark.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+val boostingStrategy = BoostingStrategy.defaultParams("Regression")
+boostingStrategy.numIterations = 50
+boostingStrategy.treeStrategy.maxDepth = 5
+boostingStrategy.treeStrategy.categoricalFeaturesInfo = Map[Int, Int]()
+val modelGB = GradientBoostedTrees.train(train, boostingStrategy)
+```
+
+The regression model options (estimating vs. classifying) will produce continuous outputs that can be used to find the right threshold. Both of these methods can be configured with tree depth and number of trees– read the Spark <a target='\_blank'  href='http://spark.apache.org/docs/1.3.0/mllib-ensembles.html'>documentation</a> for details, but general rules of thumb are the following:
+
+* Random Forest – trees are built in parallel and overtraining decreases with more trees so setting this number to be large is a great way to leverage a Hadoop environment. The max depth should be larger than GBT.
```
Comment (choukden) [inline content/blog/comparing-to-kill-a-mockingbird-to-its-sequel-with-apache-spark.md:152]: The max depth should be larger than Gradient Boosted Trees (GBT).

File: content/blog/comparing-to-kill-a-mockingbird-to-its-sequel-with-apache-spark.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+* Gradient Boosted Trees – the number of trees is directly related to overtraining and the trees are not built in parallel. This method can produce some extremely high classification rates on the training data, but set the max depth of trees to be smaller than random forest.
+
+The table below shows the commands to calculate the ROC (Receiver Operating Characteristic) for the Random Forest model – the ROC will tell the real story on the model performance.
```
Comment (choukden) [inline content/blog/comparing-to-kill-a-mockingbird-to-its-sequel-with-apache-spark.md:155]: Suggest: to calculate the Receiver Operating Characteristic (ROC) for the Random Forest model.

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Nathan,

Just some minor comments on my side. 
Dale is going to go through the editorial review process in the coming day or so.

Also, could you please add a tag for this post so it will be tied with the HPE Ezmeral Data Fabric "Platform" on HPE DEV portal:

**_hpe-ezmeral-data-fabric_**

Thanks
Denis

File: content/blog/comparing-to-kill-a-mockingbird-to-its-sequel-with-apache-spark.md (hunk 1 above)
Comment (drensing) [inline content/blog/comparing-to-kill-a-mockingbird-to-its-sequel-with-apache-spark.md:152]: Please add a comma after "with more trees"

**Review by drensing — CHANGES_REQUESTED** [review]
a very well-written post. Minimal suggestions. Very interesting post as well!

**Pull request closed without merging**